include/Gaffer/MetadataAlgo.h

```diff
@@ -100,6 +100,16 @@ bool ancestorAffectedByChange( const Plug *plug, IECore::TypeId changedNodeTypeI
 /// Copies metadata from one target to another. The exclude pattern is used with StringAlgo::matchMultiple().
 void copy( const GraphComponent *from, GraphComponent *to, const StringAlgo::MatchPattern &exclude = "", bool persistentOnly = true, bool persistent = true );
 
+///
+enum Overwrite
+{
+	KeepExisting,
+	OverwriteExisting
+};
+
+/// Copy nodule and noodle color meta data from srcPlug to dstPlug
+void copyColors( const Gaffer::Plug *srcPlug , Gaffer::Plug *dstPlug, Overwrite overwrite );
```

Comment:
OK, cool. It's not easy to squeeze it in, but we do find time for a bit of a spring clean every now and then, so do let us know if you have any bright ideas...